deno/send_message/src/functions/send_sms_twilio.ts

```diff
@@ -0,0 +1,53 @@
+export default async function (
+  variables: any,
+  phoneNumber: string,
+  message: string
+) {
+  const accountSID = variables["TWILIO_ACCOUNT_SID"];
+  const authToken = variables["TWILIO_AUTH_TOKEN"];
+  const sender = variables["TWILIO_SENDER"];
+
+  if (!accountSID) {
+    return {
+      success: false,
+      message: "TWILIO_ACCOUNT_SID is not set",
+    };
+  }
+  if (!authToken) {
+    return {
+      success: false,
+      message: "TWILIO_AUTH_TOKEN is not set",
+    };
+  }
+  if (!sender) {
+    return {
+      success: false,
+      message: "TWILIO_SENDER is not set",
+    };
+  }
+
+  try {
+    fetch(
```

Comment:
without await, this will won't throw any exception so passing an incorrect auth token will result in success.

![image](https://user-images.githubusercontent.com/1477010/206575634-47d8e0fb-902a-4ea4-8359-82071f1834c0.png)